tests/unit/client.py

```diff
@@ -589,7 +598,10 @@ def test_port_revert(self):
                 'node': 'node-01',
                 'nic': 'eth0',
                 'networks': {'vlan/native': 'net-01'}}
-        assert C.port.port_revert('mock-01', 'gi1/0/1') is None
+        response = C.port.port_revert('mock-01', 'gi1/0/1')
+        uuid_pattern = re.compile(
+            "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")
```

Comment:
We should probably factor out this regex and put it somewhere in test_common. I think it's used in some of the migration tests as well.